octodns_etchosts/__init__.py

```diff
@@ -173,7 +173,7 @@ def _apply(self, plan):
                 records.sort(key=lambda r: r._type)
 
             # Sort wildcards longest first so that we match most specific
-            self._wildcards.sort()
+            self._wildcards.sort(key=lambda w: w[0])
```

Comment:
Based on the comment [up here](https://github.com/octodns/octodns-etchosts/pull/12/files#diff-22819f4ececeb9c7336cee1fb61bf1ce4b011b621734ef59e53a6b511690babfR152-R154) I believe this needs to sort by the first 2 elements, `n` and `_type`. 

```suggestion
            self._wildcards.sort(key=lambda w: w[0:2])
```

Reply:
Yes, that is probably best. I have amended that in the previous commit